### PR TITLE
Front End Feature

### PR DESCRIPTION
**Pull request by Enoch-Chou**

### CODE REVIEW

File: project/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,18 +1,43 @@
...
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+  <body onload="getModel()">
+    <p>Gmail API Quickstart</p>
+    <!--Add buttons to initiate auth sequence and sign out-->
+    <button id="authorize_button" style="display: none;">Authorize</button>
+    <button id="signout_button" style="display: none;">Sign Out</button>
+
+    <!-- <p>Please add a question to search for information inside your inbox</p>
```
Comment (brandonvu12) [inline project/src/main/webapp/index.html]: Remove comments

Comment (Enoch-Chou) [reply project/src/main/webapp/index.html]: just did this!

File: project/src/main/webapp/gmail.js [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+        }
+        emailBodyValue = emailBodyValue.substring(200);
+        const emailDateStartIndex = decodedEmail.indexOf("Date:");
+        const emailDateEndIndex = decodedEmail.substring(emailDateStartIndex).indexOf("Message-ID");
+        var emailDateValue = decodedEmail.substring(emailDateStartIndex, emailDateStartIndex + emailDateEndIndex);
+        const MessageIDStartIndex = decodedEmail.indexOf("Message-ID:") + 12;
+        const MessageIDEndIndex = decodedEmail.substring(MessageIDStartIndex).indexOf("Subject:") + MessageIDStartIndex;
+        var messageID = decodedEmail.substring(MessageIDStartIndex, MessageIDEndIndex);
+
+        //Grabs the date of the email.
```
Comment (brandonvu12) [inline project/src/main/webapp/gmail.js]: Do you need a space after the slashes?

File: project/src/main/webapp/gmail.js [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        if (emailDateValue.indexOf("X-Notifications") != -1) {
+            emailDateValue = emailDateValue.substring(0, emailDateValue.indexOf("X-Notifications"));
+        }
+
+        //A condition to check if it's an actual email.
+        if (!hasFormatFlowed && emailDateValue.length > 1 && emailBodyValue.indexOf("Daily Insider") == -1 /*(!(messageID in emailObject))*/) {
```
Comment (brandonvu12) [inline project/src/main/webapp/gmail.js]: Remove comments

**Review by brandonvu12 — APPROVED** [review]

File: project/src/main/webapp/main.js [hunk 4]
```diff
@@ -1,5 +1,10 @@
-/** Initialize mlClass outside to have access to class outside of function */
-let mlClass;
-window.onload = function() {
-    mlClass = new MLModelEmailParse();
+function main() {
```
Comment (eswarty) [inline project/src/main/webapp/main.js]: Rename to `search` or `onClick` or something

**Review by eswarty — APPROVED** [review]

**Pull request merged**